charts/kube-prometheus-stack/values.yaml

```diff
@@ -1608,13 +1608,15 @@ prometheusOperator:
     tag: v0.50.0
     sha: ""
 
-  ## Set the prometheus config reloader side-car CPU limit
+  ## Set the prometheus config reloader side-car CPU resources
```

Comment:
As we break this already i would use the chance to do it like this:

```
  ## Prometheus-config-reloader
  ##
  prometheusConfigReloader:
    # image to use for config and rule reloading
    image:
      repository: quay.io/prometheus-operator/prometheus-config-reloader
      tag: v0.50.0
      sha: ""

    # resource config for prometheusConfigReloader
    resources:
      requests:
        cpu: 100m
        memory: 50Mi
      limits:
        cpu: 100m
        memory: 50Mi
```

deployments.yaml needs to be adjusted acordingly.